### ISSUE TITLE
Displaying the name of the sender on the message

### ISSUE THREAD

**Issue by rajaybitter** [timigod/android-chat-ui#3]

**Comment by rajaybitter** [comment]
It would be really good to have the capability, especially if you decide to implement group chats

**Comment by timigod** [comment]
Yes, this is a feature that we plan to implement, if you want to you can create a pull request.

**Comment by andkishor** [comment]
Hi, I am not able to access the constructor  "ChatMessage(String message, long timestamp, Type type, String sender)" for displaying sender name in group chat. Please reply asap.Thanks in advance

**Comment by MichaelObi** [comment]
Hi @andkishor,

Sorry for the delayed response. The fix has been made to the master branch. We'll do some reconciliation for the dev and master branches and tag a new release containing these fixes.

**Comment by swapnil591** [comment]
how to change color of sender text
